src/Compilers/CSharp/Portable/Binder/SimpleLocalScopeBinder.cs

```diff
@@ -21,5 +22,15 @@ protected override ImmutableArray<LocalSymbol> BuildLocals()
         {
             return _locals;
         }
+
+        internal override ImmutableArray<LocalSymbol> GetDeclaredLocalsForScope(CSharpSyntaxNode scopeDesignator)
```

Comment:
If it is dead code, can you please remove it? (I thought it was a common base class; my comment makes more sense for `LocalScopeBinder`)